vars/capsulePipelineWrapper.groovy

```diff
@@ -103,6 +115,83 @@ def call() {
                     sh "vegacapsule network stop --nodes-only --home-path ${env.CONFIG_HOME}"
                 }
             }
+            stage('Update networks configs') {
+                environment {
+                    TEMPLATES_HOME = "./networks-internal/stagnet3/vegacapsule/config"
+                }
+                when {
+                    expression {
+                        params.REGENERATE_CONFIGS
+                    }
+                }
+                steps {
+                    script {
+                        [
+                            'remote': '--update-network',
+                            'local': '--out-dir ./networks-internal/stagnet3/live-config'
```

Comment:
Let's do it in a different order:
- first try to do it locally,
- then remote.

Reason: treat `local` iteration as a sanity check if the config is correct. If the local iteration breaks, then we are in a safe spot: nothing was run on remote machines, and we will not upload to S3.

Current situation: if the setup is broken, the change will be partially applied to remote machines, which will be in the wrong state and inconsistent with s3.